plugins/modules/junos_package.py

```diff
@@ -174,7 +176,6 @@ def main():
         no_copy=dict(default=False, type="bool"),
         validate=dict(default=True, type="bool"),
         force=dict(type="bool", default=False),
-        transport=dict(default="netconf", choices=["netconf"]),
```

Comment:
Same as above

Reply:
This `transport` option not used anywhere.